hack/setup_env.sh

```diff
@@ -51,6 +51,19 @@ function awscli() {
   ./aws/install && rm -f /tmp/awscliv2.zip
 }
 
+function test_tools() {
+  go get github.com/onsi/ginkgo/ginkgo@v1.16.4 \
+      github.com/golang/mock/mockgen@v1.5.0 \
+      github.com/vektra/mockery/.../@v1.1.2 \
+      gotest.tools/gotestsum@v1.6.3 \
+      github.com/axw/gocov/gocov \
+      github.com/AlekSi/gocov-xml@v0.0.0-20190121064608-3a14fb1c4737
+}
+
+function linter() {
+  curl --retry 5 -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -d -b $(go env GOPATH)/bin v1.37.0
```

Comment:
it's already part of the image: https://github.com/openshift/assisted-service/blob/9d21fc4e3d71c4dae59c534e363a233fc68c9df4/Dockerfile.assisted-service-build#L5
pay attention it's a redundant step and we previously removed this logic because it [brought some flakes](https://github.com/openshift/assisted-service/pull/4077)

Reply:
v1.36.0 is very old version. It could be the reason. I had this issue.
anyway, It should be part of the logic and shared between the skipper image and the CI.
If it will be flake need to copy it from the image in the setup_env code so it can be shared.
Using and build a huge image slows down our env and not so simple and clear.

Reply:
The new linter complains about some errors so we should try to upgrade and fix the those issues

Reply:
it's not connected to the version used, it's github's rate-limiter
``quay.io`` on contrast doesn't have a rate-limiter, so it's a more stable way to go

Reply:
reverted the linter